sklearn/gaussian_process/_gpc.py

```diff
@@ -436,11 +440,11 @@ def _posterior_mode(self, K, return_temporaries=False):
             W_sr = np.sqrt(W)
             W_sr_K = W_sr[:, np.newaxis] * K
             B = np.eye(W.shape[0]) + W_sr_K * W_sr
-            L = cholesky(B, lower=True)
+            L = cholesky(B, lower=True, check_finite=False)
```

Comment:
With a similar argument as my other comment, `W_sr_K` may not be finite because it is a product of `W_sr[:, np.newaxis] * K`.